mvg/mvg.py

```diff
@@ -808,16 +809,27 @@ def list_tabular_measurements(
 
         resp_first = response.json()
         all_measurements = resp_first["items"]
-        num_v = resp_first["total"]
-        if not params:
+        total_num = resp_first.get("total", None)
+        request_limit = resp_first.get("limit", None)
+
+        has_missing_items = False
+        if not start_timestamp and not end_timestamp and total_num and request_limit:
```

Comment:
Why do we need to also add `start_timestamp` and not `end_timestamp` to the condition? I think the backend does not yet solve the interdependency between the pagination parameters and (`start_timestamp` and `end_timestamp`)

Reply:
I guess I can delete those: I initially tried to check this way if I can expect 'total' and 'limit' to not be None, but check for wether total_num is None is enough now.